angularSTEP/src/app/home/home.component.html

```diff
@@ -1,12 +1,61 @@
 
-
-<h3 (click)="logout()">Welcome {{username}}!</h3>
-<div><app-profile-card *ngFor="let user of users | async"
-    [user]='user'></app-profile-card>
+<div class="welcome-page-container">
+    <!-- this image is completely free to use with no attribution from https://www.pexels.com/photo/top-view-of-food-1640772/ -->
+    <img class='food-image' src='assets/images/food-homepage.jpg'>
+    <mat-card class="welcome-card">
+        <mat-card-title>Welcome to <b>Basil</b></mat-card-title>
+        <mat-card-subtitle>Three reciPEAS in a STEP Pod</mat-card-subtitle>
+        <mat-divider></mat-divider><br>
+        <mat-card-content><h3><b>Basil</b> is a cooking recipe website that allows cooks to find homemade/existing recipes and 
+            get the ingredients they need to make homemade meals in an easy to navigate flow without the exesive blog information! ALL while being sheltered in place! <br/>
+            </h3>
+        </mat-card-content>
+        <mat-card-actions>
+            <button mat-stroked-button color="primary" routerLink="/discover">Find Recipes!</button>
+            <button mat-stroked-button color="accent" routerLink="/upload-recipe">Upload your own recipe!</button>
+        </mat-card-actions>
+   
+    </mat-card>
+</div>
+<mat-divider></mat-divider>
+<div class="all-content">
+    <div class="content-container">
+        <div class="widget-header">
+            <h1>Top Recipes</h1>
+            <a>See More</a><br>
```

Comment:
I assume you're planning to add an href later?

Reply:
Yes I am, right now the Discover page isn't in main so I have nothing top link to. I will add a comment.